scripts/900_cleanup.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
-yum clean all && rm -rf /tmp/* /var/tmp/*
+yum clean all
+rm -rf /tmp/* /var/tmp/*
 
-grep -v $HOSTNAME /etc/hosts >/tmp/hosts && mv --force /tmp/hosts /etc/hosts
+if command -v hostname; then
+	sed --expression "/$(hostname)/ d" /etc/hosts >/tmp/hosts
```

Comment:
I am not sure how this supposed to work with lowercase env var. Does sed support auto replacing of hostname?

On the other the reasoning behind these lines were to clean the hostname if it exists to prepare for the 
actual usage of the AMI.

I don't see a necessity to check if hostname command exists or not

Reply:
> I am not sure how this supposed to work with lowercase env var. Does sed support auto replacing of hostname?

It executes hostname program, not a variable substitution. sed expression is simply deleting any lines that contain the hostname.

> I don't see a necessity to check if hostname command exists or not

It doesn't exist in `amazonlinux` docker image.

Reply:
> It executes hostname program, not a variable substitution.

My bad, sorry :)